build.ps1

```diff
@@ -1,3 +1,5 @@
+#Requires -Modules BuildHelpers,psake,PowerShellBuild
```

Comment:
This `#requires` breaks the build because these modules are not installed on the GitHub Action runners. Can you remove it please? In `./build.ps1` there is a `-Bootstrap` switch which will take care of installing them and is used when run in GH Actions. If you were running `./build.ps1` locally and didn't have these modules, you need to add the `-Bootstrap` switch.

Reply:
I see 👍 should be fixed now